Quiz Master/src/navigation/nav.tsx

```diff
@@ -0,0 +1,44 @@
+import { createBrowserRouter, RouterProvider } from "react-router-dom";
+
+import ScreenTheFirst from "../screens/options/ScreenTheFirst";
+import ScreenTwo from "../screens/quiz/ScreenTwo";
+import ThirdScreen from "../screens/results/ThirdScreen";
+import FinaleScreen from "../screens/statistics/FinaleScreen";
+import ErrorPage from "../screens/ErrorPage";
+import App from "../App";
+import { Provider } from "react-redux";
+import { store } from "../store";
+import AudioPlayer from "../components/AudioPlayer";
+import { Routes } from "../constants/routes";
+
+const router = createBrowserRouter([
+  {
+    path: Routes.Home,
+    element: <App />,
+    errorElement: <ErrorPage />,
+    children: [
+      { index: true, element: <ScreenTheFirst /> },
+      {
+        path: Routes.Quiz,
+        element: <ScreenTwo />,
```

Comment:
Here I would suggest going with a better naming. So instead of ScreenTwo, ScreenThree I would've gone with QuizScreen, ResultsScreen and so on

Reply:
The page naming was a joke, I'll update them 🤣